diff --git a/yggdrasil_decision_forests/learner/decision_tree/BUILD b/yggdrasil_decision_forests/learner/decision_tree/BUILD
index 9c021b04..0a8746a0 100644
--- a/yggdrasil_decision_forests/learner/decision_tree/BUILD
+++ b/yggdrasil_decision_forests/learner/decision_tree/BUILD
@@ -63,7 +63,7 @@ cc_library_ydf(
         "@com_google_absl//absl/time",
         "@com_google_absl//absl/types:optional",
         "@com_google_absl//absl/types:span",
-        "@eigen_archive//:eigen3_internal",
+        "@eigen_archive//:eigen3",
     ] + select({
         "//conditions:default": [
         ],
@@ -150,7 +150,7 @@ cc_library_ydf(
         "@com_google_absl//absl/time",
         "@com_google_absl//absl/types:optional",
         "@com_google_absl//absl/types:span",
-        "@eigen_archive//:eigen3_internal",
+        "@eigen_archive//:eigen3",
     ] + select({
         "//conditions:default": [
         ],
